### PR TITLE
Add Fibonacci Recursive and Iterative

### PR DESCRIPTION
**Pull request by ms10335**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: fibonacci
🏅 Granted 6 XP for @ms10335

**Comment by ziobron** [comment]
⏱ 3 XP extra for @ms10335 for on-time delivery

File: homework/fibonacci/fibonacci.hpp [hunk 1]
```diff
@@ -2,10 +2,32 @@
 
 int fibonacci_iterative(int sequence) {
     // TODO: Your implementation goes here
-    return 0;
+
+    if(sequence < 2)
+    {
+        return sequence;
+    }
+    // 0 1 1 2 3 5 8 13 21 34
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:10]: remove

File: homework/fibonacci/fibonacci.hpp [hunk 2]
```diff
@@ -2,10 +2,32 @@
...
     // TODO: Your implementation goes here
-    return 0;
+
+    if(sequence < 2)
+    {
+        return sequence;
+    }
+    // 0 1 1 2 3 5 8 13 21 34 
+    int first =1;
+    int second =1;
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:12]: Add space between operations like '+' '-' '!=' '==' '%' etc...

File: homework/fibonacci/fibonacci.hpp [hunk 3]
```diff
@@ -2,10 +2,32 @@
...
     // TODO: Your implementation goes here
-    return 0;
+   if(sequence < 2)
+   {
+       return sequence;
+   }
+   else{
+       return fibonacci_recursive(sequence -1) + fibonacci_recursive(sequence -2);
+    
+   }
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:31]: If you return variable here, you don't need to use `else if` or `else`

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**